pkg/mfa/mfa_test.go

```diff
@@ -19,20 +19,20 @@ func TestMfaEnrollFactors(t *testing.T) {
 	}
 	SetAPIKey("test")
 
-	expectedResponse := EnrollResponse{
+	expectedResponse := Factor{
 		ID:        "auth_factor_test123",
 		CreatedAt: "2022-02-17T22:39:26.616Z",
 		UpdatedAt: "2022-02-17T22:39:26.616Z",
 		Type:      "generic_otp",
 	}
-	enrollResponse, err := EnrollFactor(context.Background(), GetEnrollOpts{
+	Factor, err := EnrollFactor(context.Background(), EnrollFactorOpts{
```

Comment:
`Factor` in this case should be `factor` since this is a variable, not the Type.

```suggestion
	factor, err := EnrollFactor(context.Background(), EnrollFactorOpts{
```

Reply:
Yup these are find and Replace issues, fixing now 😬 Sorry.